### PR TITLE
bump dcos-log sha

### PR DESCRIPTION
**Pull request by mnaboka**
## High-level description

- fix bugs found during the testing
- introduce new features to download and browse files in the sandbox

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-1953](https://jira.mesosphere.com/browse/DCOS_OSS-1953)

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-1963](https://jira.mesosphere.com/browse/DCOS_OSS-1963)

## Checklist for all PRs

  - [X] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [X] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [X] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [X] Change log from the last version integrated (this should be a link to commits for easy verification and review): [dcos-log](https://github.com/dcos/dcos-log/compare/0bef08846a586a4c8b67e405f260e0d9ae845200...062a7ec95dfb70556708f73541995a0c0d356412)
  - [X] Test Results: [jenkins](https://jenkins.mesosphere.com/service/jenkins/job/dcos-cluster-ops/job/public-dcos-log-pulls/184/console)
  - [X] Code Coverage (if available): [jenkins](https://jenkins.mesosphere.com/service/jenkins/job/dcos-cluster-ops/job/public-dcos-log-pulls/184/console)
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

File: packages/dcos-integration-test/extra/test_dcos_log.py [hunk 1]
```diff
@@ -356,6 +359,9 @@ def test_log_v2_pod_logs(dcos_api_session):
         check_response_ok(response, {})
         assert 'STDERR_LOG' in response.text, "Expect STDERR_LOG in stdout file. Got {}".format(response.text)
 
+        _browse_files(dcos_api_session, pod_id, ['stdout', 'stderr', 'foo'])
+        _download_files(dcos_api_session, pod_id, ['stdout', 'stderr', 'foo'])
```
Comment (branden) [inline packages/dcos-integration-test/extra/test_dcos_log.py]: It looks like these functions are asserting some fact about the logs API. Can we name these functions according to that assertion, or explain what they're asserting in their docstrings? I can't tell what `_browse_files()` is guaranteeing from its name, and it doesn't have a docstring, so I have to read its code to know what it's verifying, which isn't ideal. A better name for `_browse_files()` might be e.g. `_assert_files_in_browse_response()`.

Comment (mnaboka) [reply packages/dcos-integration-test/extra/test_dcos_log.py]: fixed

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by amitaekbote** [comment]
looks like enterprise is failing, once enterprise PR passes I will approve that too

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by mnaboka** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by mnaboka** [comment]
@mesosphere-mergebot bump-ee

**Comment by mnaboka** [comment]
@mesosphere-mergebot label Ship It

**Pull request merged**